apps/web/README.md

```diff
@@ -34,3 +34,14 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+## Adding chains
+
+### Web
+
+Add the chain config to config/chains.ts
+Add chain name and chain id and faucetAddress to enums in types/index.ts
```

Comment:
```suggestion
Add chain name, `ChainId`, and `FaucetAddress` to enums in `types/index.ts`.
```

Nit: code highlighting, full stop, commas, match variable names to the one's used in `types/index.ts` to better reflect changes.